api/libproject/devstart.rb

```diff
@@ -1503,6 +1503,7 @@ def deploy_gcs_artifacts(cmd_name, args)
 
   Dir.chdir("cluster-resources") do
     common.run_inline(%W{./build.rb build-snippets-menu})
+    common.run_inline(%W{./build.rb generate-static-files})
```

Comment:
I would move this into `deploy_app` instead, somewhere around L1579 (before appengineStage). This copy needs to happen every time before we build the GAE app for deployment, so it should be more closely coupled to that step